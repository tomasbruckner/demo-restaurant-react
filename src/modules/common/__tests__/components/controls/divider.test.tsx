import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Divider from 'modules/common/components/controls/Divider';

describe('Divider', () => {
  it('basic example', () => {
    const tree = renderer.create(<Divider />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
