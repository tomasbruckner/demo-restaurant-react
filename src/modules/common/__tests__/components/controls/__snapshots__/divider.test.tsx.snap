// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider basic example 1`] = `
<hr
  className="MuiDivider-root"
/>
`;
